### DISCUSSION TITLE
Minimum react version

### DISCUSSION THREAD

**Discussion by jomarcel-netengine** [facebook/lexical#6320] [General]
Hey guys, does anyone know what's the minimum react version to use lexical? I'm currently on version 16.0.0 and lexical does not appear to be working

**Comment by 2wheeh** [comment]
https://github.com/facebook/lexical/blob/7f9ba1221691e9ad5599bba89c8742137eaf4a26/packages/lexical-react/package.json#L35

`>= 17` is set as a peerDependency

**Reply by jomarcel-netengine** [reply]
Got it, thanks for confirming @2wheeh
